resources/assets/less/bem/mp-history-player-score.less

```diff
@@ -87,22 +123,10 @@
     font-weight: 700;
 
     padding-right: 20px;
-
-    display: none;
```

Comment:
this actually used to avoid overflowing the table

![](https://s.myconan.net/2018-02/chrome_2018-02-02_20-31-02.png)